activities-dependency-injection/src/worker.ts

```diff
@@ -0,0 +1,23 @@
+import { Worker } from '@temporalio/worker';
+
+// @@@SNIPSTART typescript-activity-deps-worker
+import { createActivities } from './activities';
+
+async function run() {
+  // mock db connection initialization in Worker
+  const dbConnection = async () => 'Temporal' 
```

Comment:
Would you create a database connection in a Workflow? Most database drivers I've worked with open a persistent socket connection when you connect to the database, so that sounds like the sort of side-effect thing that wouldn't work well in a Workflow.

Reply:
I don’t understand your comment, this is the worker